service/src/test/java/bio/terra/cbas/common/TestMethodUtil.java

```diff
@@ -0,0 +1,122 @@
+package bio.terra.cbas.common;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import bio.terra.cbas.controllers.MethodsApiController;
+import bio.terra.cbas.dao.MethodDao;
+import bio.terra.cbas.dao.MethodVersionDao;
+import bio.terra.cbas.dao.RunSetDao;
+import bio.terra.cbas.dependencies.dockstore.DockstoreService;
+import bio.terra.cbas.dependencies.wes.CromwellService;
+import bio.terra.cbas.model.PostMethodRequest;
+import bio.terra.dockstore.client.ApiException;
+import bio.terra.dockstore.model.ToolDescriptor;
+import java.io.UnsupportedEncodingException;
+import java.net.MalformedURLException;
+import java.net.URISyntaxException;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.test.context.ContextConfiguration;
+
+@WebMvcTest
+@ContextConfiguration(classes = MethodsApiController.class)
+class TestMethodUtil {
```

Comment:
I'd strongly encourage adding the `@ExtendWith(MockitoExtension.class)` annotation to this class. Amongst other things, it checks (a) did all my mocked out methods actually get used, and (b) did any unmocked-out methods get called unexpectedly